api/inc/box_config.h

```diff
@@ -64,6 +65,7 @@ UVISOR_EXTERN const uint32_t __uvisor_mode;
         UVISOR_MIN_STACK(stack_size), \
         context_size, \
         __uvisor_box_namespace, \
+        NULL, \
```

Comment:
As of our offline discussion, the hooks should still go in the assembly file but not as weak. The OS is forced to call the macro you created here, but the macro just populates this symbol with something meaningful (might be also `NULL`).